Automattic/jetpack
# Social: Redirect to admin page after plugin activation

#### Changes proposed in this Pull Request:

* Redirect to the admin page of the plugin after plugin activation.
* Add a link to the admin page from the plugins page.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* First, make sure Publicize is not active (`jetpack docker wp option get jetpack_active_modules`)
* (If it is active, you can activate the plugin, click the toggle on the admin page, then deactivate the plugin again)
* Activate the plugin. Check that you automatically get redirected to the admin page of the plugin.
* Check that the toggle is activated. You can also run `jetpack docker wp option get jetpack_active_modules` to check if Publicize did get activated.
* Navigate back to the plugins page, and make sure the Settings link (see screenshot) is there and works.

<img width="238" alt="CleanShot 2022-05-31 at 15 08 36@2x" src="https://user-images.githubusercontent.com/1713699/171181043-edb270b1-d577-4ce3-830f-0436661465a2.png">
